network-test-app/app.js

```diff
@@ -0,0 +1,65 @@
+const { Client } = require("pg");
+const express = require("express");
+const app = express()
+
+const port = process.env.PORT;
+
+app.get("/", async (request, response) => {
+    pgServiceCredentials = process.env.DATABASES_FOR_POSTGRESQL_CONNECTION
+    if(!!pgServiceCredentials){
+        /* 
+            Postgres service credentials have been configured properly, 
+            continue with attempting to connect to service
+        */
+        try {
+            // Use env variables loaded from service binding to connect to our postgres instance
+            console.log("Connecting to PostgreSQL instance...");
+            
+            postgresSetup = JSON.parse(pgServiceCredentials);
+            cli = postgresSetup.cli;
+            postgres = postgresSetup.postgres;
+            cert = Buffer.from(postgres.certificate.certificate_base64, 'base64').toString('utf8');
+
+            const client = new Client({
+                user: postgres.authentication.username,
+                password: cli.environment.PGPASSWORD,
+                host: postgres.hosts[0].hostname,
+                database: postgres.database,
+                port: postgres.hosts[0].port,
+                ssl: {
+                    cert: cert,
+                    rejectUnauthorized: false,
```

Comment:
Can you set this to true? This is not secure, and we should set a good example for our users to use.